Overleaf sync and LaTeX result-plumbing follow-ups (#266)
symbiotic-engineering/MDOcean: pubs/renewable-energy-mdo/sections/formulation.tex

    In the multi-objective formulation, instead of $LCOE$, the two objectives are average electrical power $P_{\text{avg,elec}}$ and design-dependent capital cost $C_{\text{design}}$.
    This directly addresses the second and third LCOE limitations: the fixed charge rate and inaccurately-scaled cost components can no longer influence the optimum, and the minimum-capital design for a given power requirement is easily identified.
\else

>     LCOE is adopted for its simplicity, though it carries three limitations: it values every Joule equally, ignoring the temporal and spatial variation of electricity value on real grids \cite{akdemir_opportunities_2023,mccabe_system_2023}; it has high early-stage uncertainty, since only structural and PTO costs are modeled as design-dependent \cite{mccabe_development_2026}; and it is less relevant for the off-grid \emph{Powering the Blue Economy} applications \cite{livecchi_powering_2019,jenne_powering_2021} where power requirements are fixed or projects are unfinanced.

Author: back to quotes

    (2) to clarify a part of the optimization process that is critical to result quality yet often omitted;
    and (3) to illustrate how the MDO mindset systematically yields a ``clever'' formulation that overcomes common difficulties in large-scale optimization.
\else

>     Outlining the requirements and the resulting formulation logic grounds the optimization within the engineering design process, clarifies a step critical to result quality still often omitted, and illustrates how the MDO mindset yields a formulation that overcomes common large-scale optimization difficulties.

Author: undo change

Author: ```suggestion
    Outlining the requirements and the resulting formulation logic grounds the optimization within the engineering design process, clarifies a step that is often omitted despite its necessity, and illustrates how the MDO mindset yields a formulation that overcomes common large-scale optimization difficulties.
```
